oaknational/Oak-Web-Application
# fix(answer-order):  render correct  order in the CorrectAnswer component

##  Answers rendered in correct order


## Issue(s)

Fixes #1284

## How to test

1. Go to https://deploy-preview-1309--oak-web-application.netlify.thenational.academy/beta/teachers/key-stages/ks2/subjects/science/units/ecosystems-0008/lessons/how-do-we-construct-a-food-web-c4vk0c
2. Click on starter quiz
3. You should see question 6 order is correct

## Screenshots

How it used to look (delete if n/a):
![219679132-2cf836d3-e250-43fa-875e-58048241f4d5](https://user-images.githubusercontent.com/91190841/220986235-99cf0a4e-c8b0-4f7f-912a-2bd9651c04b4.png)


How it should now look:
<img width="1482" alt="Screenshot 2023-02-23 at 17 37 26" src="https://user-images.githubusercontent.com/91190841/220986014-d573fd31-37ab-4530-aa1f-f78f8a52bc5a.png">

## Checklist
